cmake/xworld3d_models.cmake

```diff
@@ -13,6 +13,7 @@ ExternalProject_Add(xworld3d_model_zoo
 ExternalProject_Get_Property(xworld3d_model_zoo SOURCE_DIR)
 
 add_custom_command(TARGET xworld3d_model_zoo POST_BUILD
+  COMMAND rm -rf models_3d
```

Comment:
don't do that. People building new 3d models will accidentally lose their progress, e.g. me.